pkg/daemon/daemon.go

```diff
@@ -102,6 +109,25 @@ func (dn *Daemon) Run(stop <-chan struct{}) error {
 	glog.Info("Starting MachineConfigDaemon")
 	defer glog.Info("Shutting down MachineConfigDaemon")
 
+	// Catch quickly if we've been asked to run once.
+	if dn.onceFrom != "" {
+		glog.V(2).Info("Running once per request")
+		// If we sense a local file has been provided ...
+		switch {
+		case strings.HasPrefix(string(os.PathSeparator), dn.onceFrom):
```

Comment:
This does mean absolute. We could support relative ... I can add that in my next update.

Reply:
either way is fine with me, but if it's just absolute we should make sure to be explicit about it, it might be a confusing error if you specify a relative file path and get told that you need to provide a file.

Reply:
re: Errors, I'm wondering if we need to validate the URI before proceeding or if we think that the errors from the calls read files/pull configs will be enough?